packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -963,13 +969,36 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   ///  * [_selectEndTo], which sets or updates selection end edge.
   ///  * [_finalizeSelection], which stops the `continuous` updates.
   ///  * [_clearSelection], which clear the ongoing selection.
+  ///  * [_selectParagraphAt], which selects an entire paragraph at the location.
   ///  * [selectAll], which selects the entire content.
   void _selectWordAt({required Offset offset}) {
     // There may be other selection ongoing.
     _finalizeSelection();
     _selectable?.dispatchSelectionEvent(SelectWordSelectionEvent(globalPosition: offset));
   }
 
+  /// Selects the entire paragraph at the `offset` location.
+  ///
+  /// If the paragraph is already in the current selection, selection won't
+  /// change. One call [_clearSelection] first if the selection needs to be
+  /// updated even if the paragraph is already covered by the current selection.
```

Comment:
Did you mean "One can call"?

Actually looking at the [style guide part about passive voice](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple), maybe phrase it more like: "Consider calling [_clearSelection]..."  Here and below where you also use "one".